copying.md

```diff
@@ -15,14 +17,12 @@ _the openage authors_ are:
 | Elias Hoigstvedt            | CapsAdmin                   | eliashogstvedt@gmail.com              |
 | Wilco Kusee                 | detrumi                     | wilcokusee@gmail.com                  |
 
```

Comment:
Why not use the same obfuscation as in https://gitlab.com/SFTtech/openage/commit/5087bed2745dfb023433117bd46de59ed5abb629